Kickstarter-iOS/Views/Cells/ActivityErroredBackingsTopCell.swift

```diff
@@ -0,0 +1,134 @@
+import KsApi
+import Library
+import Prelude
+import UIKit
+
+final class ActivityErroredBackingsTopCell: UITableViewCell, ValueCell {
```

Comment:
Is the _Top_ in `ActivityErroredBackingsTopCell` because it will appear at the top? I don't think we want to include the positioning information in the name of the cell, so we can just call it `ActivityErroredBackingsCell`.